src/swap/changehero.ts

```diff
@@ -292,15 +293,16 @@ export function makeChangeHeroPlugin(
       expirationDate: new Date(Date.now() + expirationFixedMs)
     }
 
-    return await makeSwapPluginQuote(order)
+    return order
```

Comment:
Please go through all these methods and replace the `const order = { ... }; return order` logic with just `return { ... }`.

Adding the extra `order` variable means TypeScript has less information to do inference with, and this will cause problems later. If you do `return { ... }`, though, TypeScript knows that this has to be a `SwapOrder` structure (because the function type says so), and this clue will let it do better inference / give better error messages.